apps/website/src/pages/ambassadors/index.tsx

```diff
@@ -216,7 +229,7 @@ const AmbassadorsPage: NextPage = () => {
           </div>
         </Section>
       </div>
-
+      <SubNav links={sectionLinks} className="z-20" />
```

Comment:
It looks like when we're using the default sort by option there are no headings and so there are no links. This is great, but it means we're rendering an empty subnav -- can we conditionally render it only when there are links to show?